File: CESEARS CIPHER/PYTHON/shift_cipher.py
def encrypt_caesar(message, shift):
    lower_alphabet = "abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz"
    upper_alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZABCDEFGHIJKLMNOPQRSTUVWXYZ"
    encrypted_text = ""
   
    for letter in message:
        if letter in lower_alphabet:
           
            position = 0
            for alpha_letter in lower_alphabet:
                if alpha_letter == letter:
                    break  
                position = position + 1
               
            new_position = position + shift
            encrypted_text = encrypted_text + lower_alphabet[new_position]
           
        elif letter in upper_alphabet:
            
            position = 0
            for alpha_letter in upper_alphabet:
                if alpha_letter == letter:
                    break
                position = position + 1
               
            new_position = position + shift
            encrypted_text = encrypted_text + upper_alphabet[new_position]
           
        else:
            encrypted_text = encrypted_text + letter
           
    return encrypted_text


print(encrypt_caesar("i want to marry a software engineer like myself", 10)) 
print(encrypt_caesar("HOW ARE YOU DOING TODAY!", 20))
print(encrypt_caesar("hello can you dash me one million naira", 6))

File: CESEARS CIPHER/JAVA/ShiftCipher.java
public class ShiftCipher {

public static String encryptCaesar(String message, int shift) {

String lowerAlphabet = "abcdefghijklmnopqrstuvwxyz";
String upperAlphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
String encrypted = "";

for (int count = 0; count < message.length(); count++) {
    char letter = message.charAt(count);

    if (Character.isLowerCase(letter)) {
        int position = lowerAlphabet.indexOf(letter);

        int newPosition = (position + shift) % 26;
        encrypted = encrypted + lowerAlphabet.charAt(newPosition);
    

    } 
    else if(Character.isUpperCase(letter)) {
        int position = upperAlphabet.indexOf(letter);

        int newPosition = (position + shift) % 26;
        encrypted = encrypted + upperAlphabet.charAt(newPosition);
    } 
    else {

        encrypted = encrypted + letter;
    }
    }

    return encrypted;
}

public static void main(String[] args) {

String result = encryptCaesar("the weather is so cold", 20);

System.out.println(result);
}
}
